Marlin/mesh_bed_leveling.h

```diff
@@ -41,6 +41,11 @@
     static FORCE_INLINE float get_probe_y(int8_t i) { return MESH_MIN_Y + (MESH_Y_DIST) * i; }
     void set_z(const int8_t px, const int8_t py, const float z) { z_values[py][px] = z; }
 
+    bool has_mesh() { return TEST(status, 0); }
```

Comment:
I'm always curious about how the compiler will `inline` things. Is there a general "rule" about when to use `inline` (apparently, it's only a cue to the compiler), versus `FORCE_INLINE` (which I presume is supposed to expand the function in-place wherever it's used), versus just leaving off both (and let the compiler decide)?